src/ui/Analytics/AnalyticsUtils.ts

```diff
@@ -0,0 +1,5 @@
+export function addActionCauseToList(newActionCause, actionCauseList) {
```

Comment:
A `remove` maybe ?

If this is to be part of the global API, there's a couple things needed: 

* JS doc documentation, explaining why it's needed, what it does etc.

* Some logic to be able to hook that on the global Coveo window object (not everybody is able to import function with a module bundler. To be able to do this, this would need to be added/exported through `RegisteredNamedMethods.ts`